### DISCUSSION TITLE
v1.1.4

### DISCUSSION THREAD

**Discussion by danieldanielecki** [Ditectrev/Professional-Scrum-Master-I-PSM-I-Practice-Tests-Exams-Questions-Answers#11] [Announcements]
**Full Changelog**: https://github.com/Ditectrev/Professional-Scrum-Master-I-PSM-I-Practice-Tests-Exams-Questions-Answers/compare/v1.1.3...v1.1.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Ditectrev/Professional-Scrum-Master-I-PSM-I-Practice-Tests-Exams-Questions-Answers/releases/tag/v1.1.4'>v1.1.4</a>.</em>

**Comment by pahameel** [comment]
how accurate are the questions and answers to clear the certification?

**Reply by danieldanielecki** [reply]
> how accurate are the questions and answers to clear the certification?

based on feedback of our students they're solid :)
